processing/src/main/java/io/druid/query/search/SearchQueryRunner.java

```diff
@@ -144,8 +144,8 @@ public void updateSearchResultSet(
   private static boolean isNilSelector(final DimensionSelector selector)
```

Comment:
Remove this method and call the method from `DimensionSelectorUtils` instead.

Reply:
ok. This was pulled in during conflict resolution. Someone introduced it in the master.